### ISSUE TITLE
Misprints in General info and Lecture 1

### ISSUE THREAD

**Issue by kharyuk** [oseledets/nla2021#1]
Good evening! There are few misprints that I have noticed while reading the currently available materials.

**lectures/general_info.ipynb**

1) Problem set rules <...> 2. Deadlines are . [no deadlines]
2) Projects <...> For inspiration you can checkout recent _conferences papers_ [maybe, _conference papers_ is better]

**lectures/lecture1/lecture-1.ipynb**

1) **Accuracy and memory** <...> Crucial note 2: <...> GPU/TPU different float types are _sypported_ [_supported_]
2) **How does number representation format affect training of neural networks (NN)?** <...> Affect of the real _number srepresentation_ [_numbers representation_]
3) **p-norm** <...> Infinity norm, or Chebyshev norm is defined _as the maximal element_ [if this is not an intentional misprint, then _as the element of the maximal absolute value_]
4) **What is a stable algorithm?** <...> The algorithm is called forward stable, if $$\| _alg_(x) - f(x) || \leq \varepsilon$$ [$$\| _\text{alg}_(x) - f(x) || \leq \varepsilon$$]
5) **What is a stable algorithm?** <...> The algorithm is called backward stable, if for any $x$ there is a close vector $x + \delta x$ such that $$_alg_(x) = f(x + \delta x)$$ [$$_\text{alg}_(x) = f(x + \delta x)$$]
6) **More examples of instability** <...> $SoftMax(x)_j <...>$ [\text{SoftMax}(x)_j]
7) **More examples of instability** <...> print("_Attempt imporove stability with add small constant:_ " [_Attempt to improve stability by adding a small constant:_]

Also [unnecessary python warnings can be suppressed](https://stackoverflow.com/questions/14463277/how-to-disable-python-warnings) globally:
`import warnings`
`warnings.filterwarnings('ignore')`
or locally
`import warnings`
`with warnings.catch_warnings():`
`____warnings.filterwarnings('ignore'):`
`____# code that triggers a warning`

**Comment by amkatrutsa** [comment]
In general info, "deadlines are strict", but "strict" is colored in red, therefore GitHub fails to render this word correctly. Try to use https://nbviewer.org/

**Comment by kharyuk** [comment]
According to this [Q&A](https://stackoverflow.com/questions/19746350/how-to-change-color-in-markdown-cells-ipython-jupyter-notebook), the `<font color=''>` tag is deprecated in HTML5 and also [may not work in some cases inside Markdown](https://ingeh.medium.com/markdown-for-jupyter-notebooks-cheatsheet-386c05aeebed). I have tested other variants:

1. Latex way: `$\color{red}{\text{strict}}$`.
- local machine: the word is displayed red, but font is slightly changed;
- github: displayed red, font is slightly changed, text is misaligned a little (appears lower than other words in a line);
- nbviewer: same as on a local machine.

2. Another tag: `<span style="color:red">strict</span>`
- local machine: displayed as expected;
- github: displayed in black;
- nbviewer: same as on a local machine.

Maybe one of this variants suits better.

**Comment by amkatrutsa** [comment]
Thank you! Fixed the deprecated way by the second proposed one.
